lib/htmltoword/document.rb

```diff
@@ -113,16 +113,14 @@ def transform_and_replace(source, stylesheet_path, file, remove_ns = false)
     def local_images(source)
       source.css('img').each_with_index do |image,i|
         filename = image['data-filename'] ? image['data-filename'] : image['src'].split("/").last
-        ext = ext_from_filename(filename)
+        ext = File.extname(filename).delete(".").downcase
 
         @image_files << { filename: "image#{i+1}.#{ext}", url: image['src'], ext: ext }
```

Comment:
I'm only using the actual file extension for the "Extension" property, I'm still using your translation code to turn jpg into jpeg and tif into tiff when setting the ContentType property (see line 146).